src/frequenz/sdk/actor/_power_managing/_base_classes.py

```diff
@@ -138,6 +139,8 @@ def get_target_power(
             proposal: If given, the proposal to added to the bucket, before the target
                 power is calculated.
             system_bounds: The system bounds for the batteries in the proposal.
+            must_return_power: If `True`, the algorithm must return a target power,
```

Comment:
Is there any reason not to always do this?

Reply:
This was supposed to reduce the number of requests going out to the power distributor, especially when there are multiple actors repeating their requests.  

But that just reminded me, we can't have this yet,  because requests to the microgrid service will expire, and we have to keep repeating them periodically.  There's a ticket for this #293, and once we have that,  we can enable `must_return_power` back.  For now,  I've added a commit https://github.com/frequenz-floss/frequenz-sdk-python/pull/692/commits/a8bf82d59f7869dea517257acddcc634f59054c9 that makes `must_return_power` always true.